tensorrt_llm/_torch/custom_ops/cpp_custom_ops.py

```
@@ -915,13 +915,25 @@ def _(sf: torch.Tensor):
         return torch.empty_like(sf, dtype=torch.uint8)
 
     @torch.library.register_fake("trtllm::moe_finalize_allreduce")
-    def _(input, residual, norm_weight, expanded_idx_to_permuted_idx,
-          shared_expert_output, expert_scale_factor, workspace, rank, nranks,
+    def _(input, residual, quant_out, scale_out, norm_weight, expanded_idx_to_permuted_idx,
```

Ruff (0.15.9)

[error] Function argument `input` is shadowing a Python builtin

(A002)